tests/ui/imports/issue-99695-b.edition_2018.stderr

```diff
@@ -0,0 +1,16 @@
+error[E0432]: unresolved import `self::p::nu`
+  --> $DIR/issue-99695-b.rs:18:23
+   |
+LL |     pub use self::p::{nu, other_item as _};
+   |                       ^^ no `nu` in `m::p`
+   |
+   = note: this could be because a macro annotated with `#[macro_export]` will be exported at the root of the crate instead of the module where it is defined
+help: a macro with this name exists at the root of the crate
+   |
+LL ~     use crate::nu;
```

Comment:
Ideally we'd suggest `pub use crate::nu;` here (by smh. copying over the visibility from the source `use`) but that's a separate issue that should be fixed separately imo.

Reply:
see #141134